adapters/adapters-test-utils/src/main/java/uk/gov/justice/services/adapters/test/utils/builder/MappingBuilder.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.services.adapters.test.utils.builder;
+
+import static java.text.MessageFormat.format;
+
+/**
+ * Builds RAML snippet defining mapping between action and media type
+ */
+public class MappingBuilder {
+
+    private static final String MAPPING_TEMPLATE =
+            "(mapping):\n" +
```

Comment:
I have reduced the use of "(mapping):" down to two usages one in MappingBuilder and one in ActionMapping.